packages/shopify-codemod/transforms/utils.js

```diff
@@ -1,4 +1,63 @@
 import j from 'jscodeshift';
+import chalk from 'chalk';
+
+function range(file, node, source, color, CONTEXT = 2) {
+  if (file === null || node === null || source === null || color == null) {
+    return '';
+  }
+
+  const lines = source.split(/\r\n|[\n\v\f\r\x85\u2028\u2029]/);
+
+  const sl = node.node.loc.start.line - 1;
+  const sc = node.node.loc.start.column;
+  const el = node.node.loc.end.line - 1;
+  const ec = node.node.loc.end.column;
+  const start = Math.max(0, sl - CONTEXT);
+  const end = Math.min(lines.length, el + CONTEXT);
+
+  let result = `(${file})\n`;
```

Comment:
I feel like this doesn't belong in this function given that this function is already doing a lot of work specifically to format the source code. Would also help reduce the number of params you need to pass in.